lib.sh

```diff
@@ -24,8 +24,7 @@ CUSTOM_CLOUD_CONFIG_DEFAULTS="$SCRIPT_DIRPATH/cloud-init/$OS_RELEASE_ID/cloud.cf
 # This location is checked by automation in other repos, please do not change.
 PACKAGE_DOWNLOAD_DIR=/var/cache/download
 
-# TODO: Lock down to specific version number for stability
-INSTALL_AUTOMATION_VERSION="latest"
```

Comment:
How did this ever work? As I read the `curl` script, it fails if invoked with "latest"?
https://github.com/containers/automation/blob/ed872b6427d727adf58635d3b3ae23def2bda1d0/bin/install_automation.sh#L91

(It's been a long day, I'm probably just missing something obvious because I don't know how this script is supposed to be invoked)